server/src/languageServerBase.ts

```diff
@@ -58,30 +53,43 @@ export class LanguageServerBase {
     // Tracks whether we're currently displaying progress.
     private _isDisplayingProgress = false;
     private _defaultWorkspacePath = '<default>';
+    private _configFileName: string | undefined;
```

Comment:
See note above. We need to support multiple config file names, not a single name.